Arm/Insts/DPI/Bitfield.lean

```diff
@@ -0,0 +1,83 @@
+/-
+Copyright (c) 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Shilpi Goel, Nevine
+-/
+-- For now, support only UBFM (immediate) aliased as LSR (immediate):
```

Comment:
The `exec_bitfield` function below appears to capture the whole UBFM behavior. Confining it to just one of its aliases seems unnecessarily restrictive.